Document the -nolint suffix (#22299)
mdn/content: files/en-us/mdn/writing_guidelines/howto/markdown_in_mdn/index.md

If the highlighting that you wish to use is not listed above you should markup the code block as `plain`.
Additional languages may be requested by following the [this process](https://github.com/orgs/mdn/discussions/170#discussioncomment-3404366).

### Suppressing linting

Writers can add a `-nolint` suffix to any of the language identifiers:

> ```html-nolint
> <p>
> I will not be linted.
> </p>
> ```

Reviewer: `````suggestion
````plain
```html-nolint
<p>
I will not be linted.
</p>
```
````
`````

> Code blocks like this will get appropriate syntax highlighting and will be recognized by the live sample system, but will be ignored by linters or automatic formatters like Prettier. This suffix is useful for demonstrating alternative formatting choices that would be "fixed" by a formatter, or intentionally invalid code.

Reviewer: ```suggestion
Code blocks like this will get appropriate syntax highlighting and will be recognized by the live sample system, but will be ignored by linters or automatic formatters like Prettier. This suffix is useful for demonstrating alternative formatting choices or intentionally invalid code that formatters or linters may complain about or autofix.
```

Author: FWIW I think this version "demonstrating alternative formatting choices or intentionally invalid code that formatters or linters may complain about or autofix" is less clear than the original.

Reviewer 2: You're right, I've reworded this one in a follow-up -> https://github.com/mdn/content/pull/22316 feedback welcome
